### ISSUE TITLE
Data Logger does not start

### ISSUE THREAD

**Issue by OWarneke** [PLCnext/PLCnext_RT_Datalogger#2]
The default config does not start and the Output.log shows something like this.

```
14.08.19 14:47:09.226 root                                                         ERROR - XML: Double hyphen within comment

14.08.19 14:47:09.227 root                                                         ERROR - XML: Double hyphen within comment: <!----

14.08.19 14:47:09.228 root                                                         ERROR - XML: Double hyphen within comment: <!-------The following GDS Port declaration is just an

14.08.19 14:47:09.228 root                                                         ERROR - XML: Double hyphen within comment: <!-------The following GDS Port declaration is just an

14.08.19 14:47:09.229 root                                                         ERROR - XML: Double hyphen within comment: <!-------The following GDS Port declaration is just an

14.08.19 14:47:09.230 root                                                         ERROR - XML: Comment not terminated

14.08.19 14:47:09.241 Arp.Services.DataLogger.Internal.DataLoggerDomain            ERROR - Could not process 'data-logger.config'! Configuration skipped.
14.08.19 14:47:09.436 Arp.Services.ProfiCloud.Internal.ProficloudTSDAdapter        INFO  - Could not read GDS metrics file /opt/plcnext/projects/ProfiCloud/metrics.json.

```

**Comment by OWarneke** [comment]
Solution:
delete the comment.

`<!------.... \n \n ----->`
Can not be parsed.

**Comment by OWarneke** [comment]
closed by  #8
